nomad/redash_server.hcl

```diff
@@ -48,28 +60,28 @@ job "redash-server" {
     task "redash-server" {
       driver = "docker"
       config {
-        image   = "${redash_image}"
+        image = "${image}"
         command = "/bin/bash"
         args = [
           "-c",
-          "python /app/manage.py database create_tables && python /app/manage.py users create_root ${redash_admin_email_id} ${redash_admin_username} --password ${redash_admin_password} --org default && /usr/local/bin/gunicorn -b 0.0.0.0:5000 --name redash -w4 redash.wsgi:app --max-requests 1000 --max-requests-jitter 100"
+          "python /app/manage.py database create_tables && python /app/manage.py users create_root admin@mail.com admin123 --password admin --org default && /usr/local/bin/gunicorn -b 0.0.0.0:5000 --name redash -w4 redash.wsgi:app --max-requests 1000 --max-requests-jitter 100"
```

Comment:
Nice, dette løser jo endel av de problemene vi har snakket om 👍 
Men samtidig, nå tvinges det en kjøring av `create_tables` og `create_root` hver gang tasken starter. Bash sin `&&` operator piper ikke videre hvis en prosess feiler, så dette kan vel bli problematisk neste gang tasken starter?

Hvor vanskelig er det å sette ut disse to "subtaskene" til post-tasks eller pre-tasks eksempelvis?

Reply:
Jeg lager en issue på dette, så kan vi se litt nærmere på dette i en ny oppgave.